### DISCUSSION TITLE
用“配置文件订阅” acl4ssr 一键导入配置，然后通过“规则集与策略组管理”添加“自定义规则集与策略组管理”，好像规则并不生效，求助，谢谢。

### DISCUSSION THREAD

**Discussion by smart-z** [vernesong/OpenClash#2657] [Q&A]
用“配置文件订阅” acl4ssr 一键导入配置，然后通过“规则集与策略组管理”添加“自定义规则集与策略组管理”，好像规则并不生效，求助，谢谢。

比如我自定义了 abc.com 只能走香港那一组的节点。但实际上不生效，他应用的还是 acl4ssr 的规则，我的自定义规则设置的还是“优先（覆盖）”的呢。

求解，谢谢

**Comment by vernesong** [comment]
看一下配置文件里面有没有写入规则
